test/test_jit_fuser.py

```diff
@@ -553,7 +553,7 @@ def fn_test_scalar_arg_requires_grad(x, p):
                                                                   "aten::_size_if_not_equal"))
 
     @unittest.skipIf(IS_SANDCASTLE, "NYI: fuser CPU support for Sandcastle")
-    @unittest.skipIf(GRAPH_EXECUTOR != ProfilingMode.LEGACY, "broken with profiling on")
+    @unittest.skip("deduplicating introduces aliasing in backward graph's outputs")
```

Comment:
This is when it would be nice to have a global "functional" context, because backward graphs are completely functional so we should be okay introducing aliasing here.